src/test/groovy/sdkman/steps/installation_steps.groovy

```diff
@@ -96,3 +96,15 @@ And(~'^I have configured "([^"]*)" to "([^"]*)"$') { String configName, String f
 And(~/^the exit code is (\d+)$/) { Integer rc ->
 	assert bash.getStatus() == rc
 }
+
+And(~'^the response headers file is created for candidate "([^"]*)" and version "([^"]*)"$') { String candidate, String version ->
+	def headersFile = "${metadataDir}/${candidate}-${version}.headers" as File
+	if (!headersFile.exists()) println bash.output
```

Comment:
The printing of the bash output probably shouldn't happen here. I think it is already done elsewhere in the bash environment helper.

Reply:
removed